README.md

```diff
@@ -14,7 +14,7 @@ Follow the videos in this and next lessons to implement the page block by block
 - Start writing styles in the `main.scss` file. It is be explained in the `Sass` lesson
 - **DON'T** try to do it `Pixel Perfect` - implement it the most `simple` way so it looks similar;
 - When done check yourself using the [CHECKLIST](./checklist.md) when finished;
-- Deploy and create a Pull Request with a [DEMO LINK](https://<your_account>.github.io/layout_landing-page/)
+- Deploy and create a Pull Request with a [DEMO LINK](https://OlhaLishenko.github.io/layout_landing-page/)
```

Comment:
Add the demo link to the PR description 
<img width="914" alt="image" src="https://github.com/user-attachments/assets/fabdca00-7910-41f9-b92d-e26c4b288460">